test/engine-luatest/pagination_test.lua

```diff
@@ -19,9 +19,11 @@ local function start_server(server)
     end)
 end
 
+-- TODO(gh-7943): Enable Vinyl cache
 tree_g.before_all(function(cg)
     cg.server = server:new{
         alias   = 'default',
+        box_cfg = {vinyl_cache = 0},
```

Comment:
I just merged the fix for #7943. This isn't longer needed.

Reply:
Removed